build.sbt

```diff
@@ -115,7 +116,7 @@ lazy val core = crossProject(JVMPlatform, JSPlatform)
 lazy val coreJVM = core.jvm
 lazy val coreJS = core.js
 
-lazy val typesafeConfig = "com.typesafe" % "config" % "1.2.1"
+lazy val typesafeConfig = "org.ekrich" %% "sconfig" % "0.7.0-SNAPSHOT"
```

Comment:
I think it would be best to create a new module `sconfig` and keep the current `typesafeConfig` module unchanged.

Reply:
Agreed. I'll change that once I make a first JVM release. Should I leave the original?